### ISSUE TITLE
Feeds' items backup

### ISSUE THREAD

**Issue by ahmed-tea** [brief-rss/brief#474]
FireFox 66.0.5
Brief 2.6
How can i make a backup from the feeds' items or export them ?

**Comment by n8v8R** [comment]
the actual feeds items are stored in a sqlite DB that resides in the profile folder

**Comment by ahmed-tea** [comment]
@n8v8R 
I want to export feeds items, uninstall the brief, install it again, and import the exported feeds items into new brief

**Comment by n8v8R** [comment]
For im/exporting _subscription list_ Brief provides that functionality (*.opml) but not for the actual feed items.
As mentioned previously actual feeds items are stored in a sqlite DB that resides in the profile folder -> https://github.com/brief-rss/brief/blob/master/modules/database.js

Not sure whether that DB is retained when Brief is getting uninstalled and neither whether/how migration of that DB would work.

**Comment by Sxderp** [comment]
No. Database is not retained on uninstall. That's a "feature" of webextensions.
